docs/playbooks/README.md

```diff
@@ -216,19 +219,130 @@ Target hosts:
+HANA nodes. The minimal configuration example is pre-populated in the supplied
+vars file `./ansible/playbooks/vars/hana_vars_example.yaml`. This file can be
+used as a template for `./ansible/playbooks/vars/hana_vars.yaml`, which must be
+present! The vars is sourced by various playbooks and contains more fields
+than are strictly necessary for just installing HANA. However, this vars file
+is sourced by multiple playbooks and enables system replication and clustering.
+By having the HANA vars in a sinle file, consistency is assured across the 
+playbooks that rely on HANA related variables.
+
+## sap-hana-system-replication
+
+Target hosts:
+
+* hana
+
+Mandatory variables:
+
+sap_hana_install_software_directory
 sap_hana_install_master_password:
 sap_hana_install_sid:
 sap_hana_install_instance_number:
+sap_domain:
+primary_site:
+secondary_site:
 
 Variable Sources:
 
-* ./ansible/playbooks/vars/sap-hana-install_vars.yaml
+* ./ansible/playbooks/vars/hana_vars.yaml
 
-The sap-hana-install playbook, like the sap-hana-storage playbook, uses an
-external role.  The playbook consumes the role to install HANA on both
-HANA nodes.  The minimal configuration is pre-populated in the supplied
-vars file `./ansible/playbooks/vars/sap-hana-install_vars.yaml`.  It should
-not be necessary to alter the file unless there is a special need, such as
-testing specific SID names, instance numbers, installing in non-default
-locations etc.
+The sap-hana-system-replication playbook cnfigures SAP system replication 
+across two HANA nodes.  Like the sap-hana-install playbook, it requires
+the `./ansible/playbooks/vars/hana_vars.yaml` vars file.  The playbook
+will ensure backups exists of all primary databases and then configure
+HANA System Replication.  Again, this playbook uses an external role
+provided by the SAP Linux Lab.
+
+## sap-hana-system-replication-hooks
+
+Target hosts:
+
+* hana
+
+Mandatory variables:
+
+sap_hana_install_software_directory
+sap_hana_install_master_password:
+sap_hana_install_sid:
+sap_hana_install_instance_number:
+sap_domain:
+primary_site:
+secondary_site:
+
+Variable Sources:
+
+* ./ansible/playbooks/vars/hana_vars.yaml
+
+Following the system replication playbook, it is necessary to install and
+configure the system replication hooks. This playbook performs the steps
+required to ensure the hooks are installed and that sudo is correctly
+configured.  Like the previous two playbooks, this one also uses the
+`hana_vars.yaml` vars file for consistency.
+
+## sap-hana-cluster
+
+Target hosts:
+
+* hana
+
+Mandatory variables:
+
+sap_hana_install_software_directory
+sap_hana_install_master_password:
+sap_hana_install_sid:
+sap_hana_install_instance_number:
+sap_domain:
+primary_site:
+secondary_site:
+
+Variable Sources:
+
+* ./ansible/playbooks/vars/hana_vars.yaml
+
+The sap-hana-cluster playbook is a complicated one. The playbook can currently
+create clusters two two fencing types: SBD and native fencing. The table
```

Comment:
I guess the first `two` is actually `to`, but even then I don't like the wording. I suggest:

> The playbook can currently create clusters using either of two fencing types: SBD and native fencing.